app/components/post_body_additional_content/index.js

```diff
@@ -33,18 +33,27 @@ function makeMapStateToProps() {
 
     return function mapStateToProps(state, ownProps) {
         const config = getConfig(state);
+        const post = getPost(state, ownProps.postId);
         const link = getFirstLink(ownProps.message);
 
         // Link previews used to be an advanced settings until server version 4.4 when it was changed to be a display setting.
         // We are checking both here until we bump the server requirement for the mobile apps.
         const previewsEnabled = getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, `${ViewTypes.FEATURE_TOGGLE_PREFIX}${ViewTypes.EMBED_PREVIEW}`) ||
             getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.LINK_PREVIEW_DISPLAY, true);
 
+        let openGraphData;
+        if (post && post.opengraph_data && post.opengraph_data.length) {
+            openGraphData = post.opengraph_data[0];
```

Comment:
I saw this on webapp too should we make it an object if we expect only one opengraph_data at once?